docs/en/aws-deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -295,6 +296,13 @@ If the Elastic Serverless Forwarder is attached to a VPC, you need to https://do
 
 NOTE: Refer to the {cloud}/ec-traffic-filtering-vpc.html[AWS PrivateLink traffic filters] documentation to find your VPC endpoint ID and the hostname to use in the `config.yml` in order to access your Elasticsearch cluster over PrivateLink.
 
+[discrete]
+[[aws-serverless-forwarder-deploy-terraform]]
+== Deploy Elastic Serverless Forwarder from Terraform
+
+The terraform files to deploy ESF can be found in https://github.com/elastic/terraform-elastic-esf[`esf-terraform` repository]. There are two requirements to deploy these files: https://curl.se/download.html[curl] and https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli[terraform]. Refer to the https://github.com/elastic/terraform-elastic-esf/blob/main/README.md[README file] to learn how to use it.
```

Comment:
I am leaving this section purposely short. There is documentation in the [README file](https://github.com/elastic/terraform-elastic-esf) and I want to avoid having to maintain two different documents for the same thing.

In the future, I would suggest creating a tutorial documentation on how to get logs similarly to [ESF tutorial](https://www.elastic.co/guide/en/observability/8.13/monitor-aws-esf.html).